paasta_tools/cli/cmds/status.py

```diff
@@ -809,6 +817,47 @@ def _print_flink_status_from_job_manager(
         dashboard_url = metadata["annotations"].get("flink.yelp.com/dashboard_url")
         output.append(f"    URL: {dashboard_url}/")
 
+    cluster_without_pnw_dash = cluster.replace("pnw-", "")
```

Comment:
this is somewhat brittle and will not work for things like infrastage

i think we might want a mapping somewhere of cluster -> ecosystem + cluster -> region (perhaps system paasta config if there's not already something for this)

Reply:
We have a very small amount of services running on (non) pnw clusters so thought I would omit adding for these cases. But have added to fully work for all mappings
https://github.com/Yelp/paasta/pull/4063/commits/56cdcc5c1d1fd0713c3788a8e309e4fb79af722f